sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ReceiveMessageBatchSyncSample.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
ReceiveMessageSyncSample is sufficient. We are returning a collection, the batch is implied by the return type.